mips/pmap.c

```diff
@@ -347,7 +348,7 @@ void tlb_exception_handler(exc_frame_t *frame) {
       log("Address %08lx not mapped by any active pmap!", orig_vaddr);
       goto fault;
     }
-    if (is_valid(pmap->pde[index])) {
+    if (pmap->pde && is_valid(pmap->pde[index])) {
```

Comment:
I've met with at `pmap` that had it's fields set to 0 at the point of this `if`. This happened on an (intentionally) invalid memory access into user memory, which triggered a recursive TLB exception. I'll see if I can trigger this again to investigate why was the pmap empty.